prisma/schema.prisma

```diff
@@ -9,6 +9,7 @@ datasource db {
 
 model User {
   id    Int     @id @default(autoincrement())
-  email String  @unique
+  email String  
   name  String?
```

Comment:
Why did you make it optional? I think every user must have a name and a password